staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -628,22 +628,24 @@ type ClockOffsetUTC struct {
 
 // ClockOffsetTimezone sets the guest clock to the specified timezone.
 // Zone name follows the TZ environment variable format (e.g. 'America/New_York').
-// ---
+//
 // +k8s:openapi-gen=true
 type ClockOffsetTimezone string
 
 // Represents the clock and timers of a vmi.
-// ---
+//
 // +k8s:openapi-gen=true
+// +kubebuilder:pruning:PreserveUnknownFields
```

Comment:
here's another seemingly random kubebuilder marker

Reply:
I will remove all occurrences of this.